src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateGlobalUsings_WebApp.cs

```diff
@@ -54,17 +46,25 @@ public void It_generates_web_imports_and_builds_successfully()
 global using global::Microsoft.Extensions.DependencyInjection;
 global using global::Microsoft.Extensions.Hosting;
 global using global::Microsoft.Extensions.Logging;
+global using global::System;
+global using global::System.Collections.Generic;
+global using global::System.IO;
+global using global::System.Linq;
+global using global::System.Net.Http;
+global using global::System.Net.Http.Json;
+global using global::System.Threading;
+global using global::System.Threading.Tasks;
```

Comment:
Please confirm the new order won't trigger [SA1208](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1208.md)
![image](https://user-images.githubusercontent.com/4403806/129126767-e622b44e-eae9-4dd6-9115-3960028e6c12.png)

Reply:
As long as the analyzer does not run on generated files, this shouldn't be a concern.